yoda.sh

```diff
@@ -22,6 +22,9 @@ make -j$JOBS
 make install
 )
 
+### list the content of lib directory
+ls -ld $INSTALL_ROOT/lib/*
```

Comment:
`ls -ld $INSTALLROOT/lib`

Reply:
thanks @ktf , my bad.
Although I fear that missing * will not list the files in the directory